apps/gallery/index.html

```diff
@@ -201,7 +201,7 @@ <h1 id="edit-title" data-l10n-id="edit"></h1>
         </div>
 
         <div id="edit-enhance-options" class="edit-options-bar hidden">
-          <span id="edit-enhance-status"></span>
+          <p id="edit-enhance-status"></p>
         </div>
```

Comment:
If we keep span and update edit-enhance-options as below, we should be able to get two line-effects
#edit-enhance-options {
text-align: center;
line-height: 2.2rem;
font-size: 1.8rem;
margin: 1rem 3rem;
}